<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MasterMind</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>guess.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeHistory">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Guess</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Output</string>
     </property>
    </column>
   </widget>
   <widget class="QLCDNumber" name="lcdLives">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textChecked">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGuess">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Guess</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineGuess">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Guess here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLives">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lives</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="itemNewGame"/>
    <addaction name="separator"/>
    <addaction name="itemQuit"/>
   </widget>
   <widget class="QMenu" name="menuCheats">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Cheats</string>
    </property>
    <addaction name="itemShowGoal"/>
    <addaction name="itemInfLives"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="itemGuide"/>
    <addaction name="separator"/>
    <addaction name="itemAboutMMind"/>
    <addaction name="itemAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCheats"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="itemNewGame">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="itemQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="itemShowGoal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Goal</string>
   </property>
  </action>
  <action name="itemInfLives">
   <property name="text">
    <string>HESOYAM (inf lives)</string>
   </property>
  </action>
  <action name="itemGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="itemAboutMMind">
   <property name="text">
    <string>About MasterMind-Qt</string>
   </property>
  </action>
  <action name="itemAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>itemAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_aboutqt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemAboutMMind</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemShowGoal</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_showgoal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGuess</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnGuess_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemInfLives</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_inflives()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemNewGame</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_newgame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemGuide</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_guide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGuess</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnGuess_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonLives</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnLives_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGuess</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineGuess_txtchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_aboutqt()</slot>
  <slot>action_about()</slot>
  <slot>action_guide()</slot>
  <slot>action_showgoal()</slot>
  <slot>action_inflives()</slot>
  <slot>action_newgame()</slot>
  <slot>lineGuess_txtchange()</slot>
  <slot>btnLives_click()</slot>
  <slot>btnGuess_click()</slot>
 </slots>
</ui>
